full/Dockerfile

```diff
@@ -159,9 +159,7 @@ RUN curl -fsSL https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.t
         golang.org/x/tools/gopls@v0.5.5 && \
     go get -u -v -d github.com/stamblerre/gocode && \
     go build -o $GOPATH/bin/gocode-gomod github.com/stamblerre/gocode && \
-    rm -rf $GOPATH/src && \
-    sudo rm -rf $GOPATH/pkg && \
-    rm -rf /home/gitpod/.cache/go
+    sudo rm -rf $GOPATH/src $GOROOT/src $GOPATH/pkg $GOROOT/pkg /home/gitpod/.cache/go /home/gitpod/.cache/go-build
```

Comment:
@csweichel Please kindly confirm that all these paths are indeed safe to delete (see also https://github.com/gitpod-io/workspace-images/pull/318#issuecomment-741931074)

Reply:
From Slack:

> If we must delete something, I reckon the `/home/gitpod/.cache/go-build` dir can go.
> afaik that’s just a build cache that would be recreated